src/test/librados/asio.cc

```diff
@@ -143,7 +143,7 @@ TEST_F(AsioRados, AsyncWriteCallback)
                         success_cb);
 
   auto failure_cb = [&] (boost::system::error_code ec) {
-    EXPECT_EQ(boost::system::errc::read_only_file_system, ec);
+    EXPECT_EQ(boost::system::errc::read_only_file_system, ec.value());
```

Comment:
Nit: any particular reason why these needed to be changed?

Reply:
Boost uses those error code classes and the assertions seemed incorrect:

/mnt/ramdisk/workspace/ceph/src/test/librados/asio.cc:197
Expected equality of these values:
  boost::system::errc::read_only_file_system
    Which is: 30
  ec
    Which is: system:30
}}}

To be honest, I'm wondering how come we only hit this on Windows.

Reply:
It might not like that `errc::` -- could try dropping that.

Reply:
Double checked, that's a simple enum under the ``errc`` namespace, dropping it will lead to an error: https://github.com/boostorg/system/blob/boost-1.73.0/include/boost/system/error_code.hpp#L57

Luckily we no longer need to explicitly use ``value()`` after bumping the Boost version to 1.73. It could be that a previous version was missing a constructor or overloaded operator. Anyway, I'm dropping this commit from the PR.